### ISSUE TITLE
Unnamed tuple not auto converted to named tuple anymore

### ISSUE THREAD

**Issue by def-** [nim-lang/Nim#4649]
This used to work, but now fails:

``` nim
import tables

let ops: Table[string, tuple[prec: int, rAssoc: bool]] =
  { "^": (4, true)
  , "*": (3, false)
  , "/": (3, false)
  , "+": (2, false)
  , "-": (2, false)
}.toTable
```

```
x.nim(9, 2) Error: type mismatch: got (Table[system.string, (int, bool)]) but expected 'Table[system.string, tuple[prec: int, rAssoc: bool]]'
```

**Comment by Araq** [comment]
Yeah as I said, I am not sure what the spec should say.

**Comment by Parashurama** [comment]
This issue appear to be solved as of current devel (fd0ab1df3efa51870d6746ed0ce6479f8934369d)
